### DISCUSSION TITLE
I'm Ellie, One of a fews

### DISCUSSION THREAD

**Discussion by mdotrostami** [WeAreMahsaAmini/Core#367] [Introductions]

**Comment by Pink3lephant** [comment]
Hi @elliminium 

Nice to hear from you. We are working on a project to distribute knowledge on how to create VPN servers that actually work (think of it as meta-Outline, where you can setup several protocols). 

At the moment we are working on our landing page (Nextjs / Strapi], it's in progress but we need more resources to finish it faster. 

Can you please have a look and let me know if you're interested in helping out (both on FE and also setting up VPNs / Tutorials /...)? 

🐜  Current website: [Antyfilter.com](https://Antyfilter.com)
👨‍💻 [Github](https://github.com/Antyfilter)


You can email me at `freeinternettech` [at] gmail. 

To freedom 🕊️
